go/teams/loader.go

```diff
@@ -469,6 +476,13 @@ func (l *TeamLoader) load2(ctx context.Context, arg load2ArgT) (ret *load2ResT,
 	}
 
 	defer l.G().CTraceTimed(ctx, traceLabel, func() error { return err })()
+	defer func() {
+		if err != nil {
+			mctx := libkb.NewMetaContext(ctx, l.G())
```

Comment:
make this metacontext when entering the function, after log tags are assigned.

Reply:
Also, simplify, just do this check after the call to `load2Inner` fails, i.e., no need for a `defer`